easybuild/easyconfigs/t/torchvision/torchvision-0.13.1-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -17,7 +17,10 @@ dependencies = [
     ('CUDA', '11.7.0', '', SYSTEM),
     ('Python', '3.10.4'),
     ('Pillow-SIMD', '9.2.0'),
+    ('libjpeg-turbo', '2.1.3'),
     ('PyTorch', '1.12.0', '-CUDA-%(cudaver)s'),
 ]
 
+preinstallopts = 'WITH_CUDA=1 TORCHVISION_INCLUDE="$EBROOTLIBJPEGMINTURBO/include:$TORCHVISION_INCLUDE"'
```

Comment:
If we're going forward with https://github.com/easybuilders/easybuild-easyblocks/pull/3322 (which makes sense), we should clean this up accordingly...

Reply:
i think we can just close this PR, as libjpeg-turbo is already a dep of Pillow-SIMD?
once the easyblock is merged i'll remove the TORCHVISION_INCLUDE from the easyconfigs that have it set